src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -48,7 +48,7 @@
     /**
      * The property name of the number of seen frames to keep track of.
      */
-    public static final String SEEN_FRAME_HISTORY_SIZE_PNAME
+    private static final String SEEN_FRAME_HISTORY_SIZE_PNAME
```

Comment:
darn, this one doesn't get used anywhere externally (nor does it seem like it should be accessible).  but if that's the case i will change it back.